src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -35,13 +35,12 @@ COPY {{artifactTarball}} /tmp/kibana.tar.gz
 RUN mkdir /usr/share/kibana
 WORKDIR /usr/share/kibana
 RUN tar \
-{{#serverless}}
+  --strip-components=1 \
+  -zxf /tmp/kibana.tar.gz {{#serverless}}&& \
 # Exclude serverless.yml disabled assets
-  --exclude=screenshotting-plugin/chromium \
-  --exclude=screenshotting-plugin/server/assets \
+  rm -rf node_modules/@kbn/screenshotting-plugin/chromium && \
```

Comment:
the --exclude flag in busybox tar vs gnu tar has a different pattern format.  rm should be more consistent.